Public/Src/IDE/VsCode/BuildXL.IDE.VsCode.dsc

```diff
@@ -1,22 +1,48 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-import {Transformer} from "Sdk.Transformers";
+import {Artifact, Transformer} from "Sdk.Transformers";
 import * as ManagedSdk from "Sdk.Managed";
 import * as Deployment from "Sdk.Deployment";
 import * as Transformers from "Sdk.Transformers";
 import * as DetoursServices from "BuildXL.Sandbox.Windows";
 import * as Branding from "BuildXL.Branding";
 import * as VSIntegration from "BuildXL.Ide.VsIntegration";
+import {Node, Npm} from "Sdk.NodeJs";
 
-namespace LanguageService.Server {
+namespace VsCode.Client {
+    export declare const qualifier: {};

```

Comment:
TODO: add a comment about why all this stuff is declared in a new namespace whose qualifier is empty.

The comment should be something like: "Declaring a new namespace with empty qualifier space to make sure the values inside are evaluated only once".

If we moved all the values from this namespace into the `LanguageService.Server` namespace below (whose qualifier space is non-empty), we'd run `npm install` and `npm run compile` for each of the qualifiers (and the qualifiers on Windows are: Windows/Debug, Windows/Release, MacOS/Debug, MacOS/Release) even though the result of those `npm` operations does not depend on the current qualifier value.